test/integ/quote-xv2.test.ts

```diff
@@ -125,38 +63,72 @@ describe('quoteUniswapX-v2', function () {
 
           const order = new UnsignedV2DutchOrder((quote as any).orderInfo, 1);
           expect(status).to.equal(200);
-
+          expect(order.info.cosigner).to.equal(ethers.constants.AddressZero);
           expect(order.info.swapper).to.equal(alice.address);
           expect(order.info.baseOutputs.length).to.equal(1);
-          expect(parseInt(order.info.baseOutputs[0].startAmount.toString())).to.be.greaterThan(9000000000);
-          expect(parseInt(order.info.baseOutputs[0].startAmount.toString())).to.be.lessThan(11000000000);
-          expect(parseInt(order.info.baseInput.startAmount.toString())).to.be.greaterThan(9000000000);
-          expect(parseInt(order.info.baseInput.startAmount.toString())).to.be.lessThan(11000000000);
+        } catch (e: any) {
+          expect(e.response.status).to.equal(404);
+          expect(e.response.data.detail).to.equal('No quotes available');
+        }
+      });
+
+      // TODO: maybe move this to param-api integ tests?
```

Comment:
This is somewhat tricky for v2 because param-api always posts to order service when a quote is found. This means that we'd pollute our production DB every time the integ tests are run. We could use sepolia for integ tests but I doubt we'll get any quotes -- which I think is okay!

Reply:
Yeah theyre all in there starting here https://github.com/Uniswap/routing-api/blob/76af7c0c43110366746408f64464537ee3fb0432/test/mocha/e2e/quote.test.ts#L318